SEDaily-IOS/PodcastPageViewController.swift

```diff
@@ -46,48 +46,59 @@ class PodcastPageViewController: TabmanViewController, PageboyViewControllerData
     func loadViewControllers() {
         let layout = UICollectionViewLayout()
         
-        let child_1 = GeneralCollectionViewController(collectionViewLayout: layout, type: API.Types.new)
-        child_1.tabTitle = L10n.tabTitleAll
+        let child_1 = GeneralCollectionViewController(collectionViewLayout: layout,
+                                                      tabTitle: PodcastCategoryIds.All.description)
         viewControllers.append(child_1)
         
-        let child_2 = GeneralCollectionViewController(collectionViewLayout: layout, tagId: 1068, type: API.Types.new)
-        child_2.tabTitle = L10n.tabTitleBusinessAndPhilosophy
-        viewControllers.append(child_2)
         
-        let child_3 = GeneralCollectionViewController(collectionViewLayout: layout, tagId: 1082, type: API.Types.new)
-        child_3.tabTitle = L10n.tabTitleBlockchain
+        let child_2 = GeneralCollectionViewController(collectionViewLayout: layout,
```

Comment:
Consider not creating counting variables, e.g, _1, _2, _3...etc, rather put all your items in an array and iterate over them.